components/territory-form.js

```diff
@@ -161,6 +162,22 @@ export default function TerritoryForm ({ sub }) {
           name='moderated'
           groupClassName='ms-1'
         />
+        <BootstrapForm.Label>nsfw</BootstrapForm.Label>
```

Comment:
I'd love to see both this `nsfw` section and `moderation` sections moved to an accordian if you're up for it. It's not a big deal and if you're over it already, I can patch it in, but for this "extra" stuff I like to hide it:

![Screenshot 2024-02-08 at 4 26 26 PM](https://github.com/stackernews/stacker.news/assets/34140557/1cc51e6e-8b70-4d8a-841c-24c139f520a4)
![Screenshot 2024-02-08 at 4 26 31 PM](https://github.com/stackernews/stacker.news/assets/34140557/1f2ae76e-dc82-4b8b-8f34-0cc6d0278bd7)

Reply:
https://github.com/stackernews/stacker.news/pull/788/commits/2de4132089249885ecb763fb4da6cb13e88a012c

Closed
<img width="961" alt="Screenshot 2024-02-09 at 10 58 32 AM" src="https://github.com/stackernews/stacker.news/assets/158518982/4b770dff-9070-46fa-9b23-dccdc7485c97">

Open
<img width="763" alt="Screenshot 2024-02-09 at 10 58 46 AM" src="https://github.com/stackernews/stacker.news/assets/158518982/ad02fd1e-878a-4436-96b6-c2541ebcf7d0">